vsts/scripts/copySdksFromProdToStorageAccount.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+# --------------------------------------------------------------------------------------------
+
+set -ex
+
+declare -r REPO_DIR=$( cd $( dirname "$0" ) && cd .. && cd .. && pwd )
+source $REPO_DIR/platforms/__common.sh
+
+azCopyDir="/tmp/azcopy-tool"
+
+function blobContainerExistsInDestination() {
+	local containerName="$1"
+	local exitCode=1
+	curl -I "$destinationSdk/$containerName?restype=container" 2> /tmp/curlError.txt 1> /tmp/curlOut.txt
+	grep "HTTP/1.1 200 OK" /tmp/curlOut.txt &> /dev/null
+	exitCode=$?
+	rm -f /tmp/curlOut.txt
+	rm -f /tmp/curlError.txt
+	if [ $exitCode -eq 0 ]; then
+		return 0
+	else
+        return 1
+	fi
+}
+
+function copyBlobContainerFromProdToDestination() {
+    local platformName="$1"
+
+    if shouldOverwriteSdk || shouldOverwritePlatformSdk $platformName; then
+        echo
+        echo "Overwriting blob container '$platformName' in storage account '$destinationSdk'."
+        # azcopy copy [source] [destination] [flags]
+        "$azCopyDir/azcopy" copy \
+            "$PROD_SDK_STORAGE_BASE_URL/$platformName/$PROD_STORAGE_SAS_TOKEN" \
+            "$destinationSdk/$platformName/$sasToken" --overwrite true --recursive true
+    elif blobContainerExistsInDestination $platformName; then
+        echo
+        echo "Blob container '$platformName' already exists in storage account '$destinationSdk'. Skipping copying it..."
+    else
+        echo
+        echo "Blob container '$platformName' does not exist in storage account '$destinationSdk'. Copying it from $PROD_SDK_STORAGE_BASE_URL..."
+        # azcopy copy [source] [destination] [flags]
+        "$azCopyDir/azcopy" copy \
+            "$PROD_SDK_STORAGE_BASE_URL/$platformName/$PROD_STORAGE_SAS_TOKEN" \
```

Comment:
Same comment here about using named parameters vs. positional.

Reply:
In the off chance that there is no way to explicitly do this, I've added a dry run parameter which will allow us to run the job without uploading or downloading anything, so that we can verify that we are inputting the correct parameters beforehand